lib/fictium/configurations/api_blueprint.rb

```diff
@@ -0,0 +1,31 @@
+module Fictium
+  class Configuration
+    class ApiBlueprint
+      attr_accessor :resources_group_name, :host, :footer_header,
+                    :api_version_formatter, :terms_of_service_formatter, :license_formatter
+
+      def initialize
+        self.host = 'https://change.me.at.api_blueprint.config'
```

Comment:
:thinking: you really need this? I prefer to do something like

```rb
# lib/fictium/exporters/api_blueprint_exporter/header_formatter.rb

def initialize
  super
  self.host = host
end
```

Reply:
Only if host is given as parameter which doesn't exists.
You can override it doing:

```rb
Fictium.configure do |c|
  c.api_blueprint.host = 'other host'
end
```

in your files.